docs/reference/firmware.md

```diff
@@ -8675,12 +8711,25 @@ Parameters:
 
   * format : format string
 
-`void operator()(const char *format, ...)`  
+`void log(const char *format, ...)`  
+`void operator()(const char *format, ...)`
+
+```cpp
+// EXAMPLE
+Log("This is %s message", "info");
+```
+
+Generate log message with the default logging level (`LOG_LEVEL_INFO`).
+
+Parameters:
+
+  * format : format string
+
+`void log(LogLevel level, const char *format, ...)`  
 `void operator()(LogLevel level, const char *format, ...)`
 
 ```cpp
 // EXAMPLE
-Log("This is %s message", "info"); // Uses default logging level
 Log(LOG_LEVEL_INFO, "This is %s message", "info");
```

Comment:
Same here - let's use a different format param other than "info" to avoid confusion with logging levels.